Account page link; case of a single review/lesson

imasaru opened the pull request:
Well, here we go!

I do have one more thing though - it seems a bit cumbersome to merge a PR with commits that add something, remove it, and then add something else in its place. This is what happened with me.

What do you suggest I do? Should I remove those other commits and just push the last one, or should I keep them all for reference?

imasaru committed:
Commit 1 - Fix for case of a single lesson/review

imasaru committed:
Commit 2 - Account Page Redirect

I'm trying to implement a redirect... I can't figure out why it isn't working, though. I tried this code in the Developer Console in Vivaldi, and it worked.

imasaru committed:
Scrapped redirect (replaced w/ link); corrected nbReviews ref.

imasaru committed:
Fix for case of single review/lesson (naei implementation)

naei commented:
You already opened PR #8. Please next time avoid duplicating pull requests. If you have changes to make you can just update your PR instead of closing the first one and reopening a new one. 
To answer your question: no need to remove your old commits, I will handle this once the review approved. I will have a look at your changes in the next days. :)

imasaru committed:
Fixed clerical errors in README

imasaru commented:
Agh! Sorry, I forgot 😅 

I also added some edits that I made to the README.

imasaru committed:
Edited notification.mp3

It is a bit loud, so I adjusted it to play along better with the current system audio level.

imasaru committed:
Changed units to standard SI; edited strings

Not sure how to specify v1 without making the string look bad in the extension...
